client/src/App.tsx

```diff
@@ -1,27 +1,31 @@
 import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom";
+import Header from "./components/Header/Header";
 
 function App() {
   return (
     <BrowserRouter>
-      <Routes>
-        <Route
-          path="/"
-          element={
-            <div className="flex min-h-screen items-center justify-center">
-              <h1 className="mb-5 rounded-xl border bg-gray-200 px-16 py-8 text-3xl font-bold text-gray-700 shadow">
-                Hello World !
-              </h1>
-            </div>
-          }
-        />
-        <Route />
-        <Route />
-        <Route />
-        <Route />
-        <Route />
-        <Route />
-        <Route path="*" element={<Navigate to="/" />} />
-      </Routes>
+      <main className="min-h-screen">
+        <Header />
+        <Routes>
+          <Route
+            path="/"
+            element={
+              <div className="flex min-h-[calc(100vh-4.5rem)] items-center justify-center">
+                <h1 className="rounded-xl border bg-gray-200 px-8 py-4 text-lg font-bold text-gray-700 shadow">
+                  Hello World !
+                </h1>
+              </div>
+            }
+          />
+          <Route />
```

Comment:
Lets remove all the routes not being used

Reply:
Removed, also added a not found route, for which I created a ticket to make a page for.